package.json

```diff
@@ -30,7 +30,7 @@
     "@atomist/sdm-pack-changelog": "0.1.1-20180713073534",
     "@atomist/sdm-pack-k8": "0.1.2-20180705162518",
     "@atomist/slack-messages": "^0.12.1",
-    "@atomist/clj-editors": "0.1.2",
+    "@atomist/clj-editors": "0.3.3",
     "fs-extra": "^6.0.0",
     "lodash": "^4.17.10",
     "logzio-nodejs": "^0.4.10",
```

Comment:
That dependency doesn't belong there. It is brought in transitively form the logzio extension.